src/main/resources/templates/elements.html

```diff
@@ -0,0 +1,708 @@
+							<li><a th:href="products.html">Products</a></li>
+							<li class="menu-has-children"><a th:href="@{#}">Blog</a>
+								<ul>
+									<li><a th:href="blog-home.html">Blog Home</a></li>
+									<li><a th:href="blog-single.html">Blog Single</a></li>
+								</ul>
+							</li>
+							<li><a th:href="contact.html">Contact</a></li>
+							<li class="menu-has-children"><a th:href="@{#}">Dropdown</a>
+								<ul>
+									<li><a th:href="elements.html">Elements</a></li>
+									<li><a th:href="product-details.html">Product Details</a></li>
+									<li class="menu-has-children"><a th:href="@{#}">Level 2</a>
+										<ul>
+											<li><a th:href="@{#}">Item 1</a></li>
+											<li><a th:href="@{#}">Item 2</a></li>
+										</ul>
+									</li>
+								</ul>
+							</li>
+						</ul>
+					</nav><!-- #nav-menu-container -->
+				</div>
+			</div>
+		</header><!-- #header -->
+			  
+			<!-- start banner Area -->
+		<section class="banner-area relative" id="home">
+			<div class="overlay overlay-bg"></div>
+			<div class="container">
+				<div class="row d-flex align-items-center justify-content-center">
+					<div class="about-content col-lg-12">
+						<h1 class="text-white">
+							Blog Home
+						</h1>
+						<p class="text-white link-nav"><a th:href="index.html">Home </a>  <span class="lnr lnr-arrow-right"></span>  <a th:href="blog-home.html"> Blog Home</a></p>
+					</div>
+				</div>
+			</div>
+		</section>
+			<!-- End banner Area -->	
+
+			<!-- Start Sample Area -->
+			<section class="sample-text-area">
+				<div class="container">
+					<h3 class="text-heading">Text Sample</h3>
+					<p class="sample-text">
+						Every avid independent filmmaker has <b>Bold</b> about making that <i>Italic</i> interest documentary, or short film to show off their creative prowess. Many have great ideas and want to “wow” the<sup>Superscript</sup> scene, or video renters with their big project.  But once you have the<sub>Subscript</sub> “in the can” (no easy feat), how do you move from a <del>Strike</del> through of master DVDs with the <u>“Underline”</u> marked hand-written title inside a secondhand CD case, to a pile of cardboard boxes full of shiny new, retail-ready DVDs, with UPC barcodes and polywrap sitting on your doorstep?  You need to create eye-popping artwork and have your project replicated. Using a reputable full service DVD Replication company like PacificDisc, Inc. to partner with is certainly a helpful option to ensure a professional end result, but to help with your DVD replication project, here are 4 easy steps to follow for good DVD replication results: 
+
+					</p>
+				</div>
+			</section>
+			<!-- End Sample Area -->
+			<!-- Start Button -->
+			<section class="button-area">
+				<div class="container border-top-generic">
+					<h3 class="text-heading">Sample Buttons</h3>
+					<div class="button-group-area">
+						<a th:href="@{#}" class="genric-btn default">Default</a>
```

Comment:
you need to wrap only relative links inside th:href="@{}" construction. if the link is static you can use simple href property

Reply:
ok, I fixed